src/main/webapp/index.html

```diff
@@ -1,34 +1,114 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta
       charset="UTF-8"
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Overnightly</title>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="constant-style.css" />
     <link rel="stylesheet" href="style.css" />
     <link
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
     />
   </head>
   <body>
-    <div class="row">
-      <div class="col s12 m4">
-        <div class="card">
-          <div class="card-image">
-            <img src="assets/img/placeholder.png" />
```

Comment:
Add blank alt tag according to HTML style guide (alt="")

Reply:
this is deleted code (was just a placeholder)

Reply:
Nvm just noticed this was deleted